apitools/base/py/base_api_test.py

```diff
@@ -198,9 +198,9 @@ def fakeMakeRequest(*unused_args, **unused_kwargs):
         service = FakeService(client=client)
         request = SimpleMessage()
         with mock(base_api.http_wrapper, 'MakeRequest', fakeMakeRequest):
-            with self.assertRaises(exceptions.HttpError) as error_context:
+            with self.assertRaises(exceptions.HttpBadRequestError) as err:
                 service._RunMethod(method_config, request)
-        http_error = error_context.exception
+        http_error = err.exception
         self.assertEquals(400, http_error.status_code)
```

Comment:
You can drop this line now, right? :wink:

Reply:
Done!